include/pulsar/ConsumerConfiguration.h

```diff
@@ -554,6 +593,8 @@ class PULSAR_PUBLIC ConsumerConfiguration {
     bool isStartMessageIdInclusive() const;
 
     friend class PulsarWrapper;
+    friend class DeadLetterQueueTest;
+    friend class DeadLetterQueueTest_testWithoutConsumerReceiveImmediately_Test;
```

Comment:
Could you replace them with `PulsarFriend` and add some methods to `PulsarFriend` to access private fields of `ConsumerConfiguration`? Once the test name changed, these friend classes would be meaningless.

Reply:
Yes. This way is better.